app/src/test/java/com/example/fittracker/LogInTest.java

```diff
@@ -1,8 +1,8 @@
 package com.example.fittracker;
```

Comment:
please change the name of this class to LogInModelTest for example, because you are testing only the model here

Reply:
done